src/DynamoRevit/DynamoRevit.cs

```diff
@@ -890,4 +912,85 @@ public override IEnumerable<string> GetDynamoUserDataLocations()
             return paths;
         }
     }
+
+    [Serializable()]
+    public class ExtensionInfo
+    {
+        public string NodePath { get; set; }
+        public string LayoutSpecsPath { get; set; }
+    }
+    internal static class DynamoRevitInternalNodes
+    {
+        private static IEnumerable<string> GetAllExtensionFiles()
+        {
+            string currentAssemblyPath = Assembly.GetExecutingAssembly().Location;
+            string currentAssemblyDir = Path.GetDirectoryName(currentAssemblyPath);
+
+            string internalNodesDir = Path.Combine(currentAssemblyDir, "packages");
```

Comment:
so this assumes a packages folder inside of Dynamo For Revit? or wherever DynamoRevit.dll is loading from?

Reply:
To be consistent I would suggest putting them under `Revit\nodes` instead of under Revit folder, also can we get rid of the `packages` folder since it is now no longer a package?

Reply:
Is the folder "[Revit]\Addins\DynamoForRevit\Revit"?

Reply:
i updated it to use "[Revit]\Addins\DynamoForRevit\Revit\nodes